cloud_governance/policy/ibm/tag_baremetal.py

```diff
@@ -0,0 +1,99 @@
+
```

Comment:
I think we need to add it to documentation

Reply:
yes, ok.

Reply:
done